src/query/expression/src/types.rs

```diff
@@ -281,6 +281,7 @@ pub trait ValueType: Debug + Clone + PartialEq + Sized + 'static {
 
     fn try_downcast_scalar<'a>(scalar: &'a ScalarRef) -> Option<Self::ScalarRef<'a>>;
     fn try_downcast_column<'a>(col: &'a Column) -> Option<Self::Column>;
+    fn try_downcast_column_ref<'a>(col: &'a Column) -> Option<&Self::Column>;
```

Comment:
Column's clone is trivial, is this new method benefit lot?

Reply:
Yes, I found that some of them are cloned instead of clone.